enum_list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list.flatten
+  end
+
+  def each(&block)
+    @list.each(&block)
+    self
+  end
+end
```

Comment:
- [ ] As the previous reviewer mentioned you should implement your own `#each` method, not use the ruby method